tensorflow_probability/examples/jupyter_notebooks/Fitting_DPMM_with_Symmetric_Dirichlet_Prior_Using_pSGLD.ipynb

```diff
@@ -0,0 +1,975 @@
+      ]
+    },
+    {
+      "metadata": {
+        "id": "A7RysXxCBNS_",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "## Samples"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "k-aVH4b4Bxzo",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "First, we set up a toy dataset. We generate 50,000 random samples from three bivariate Gaussian distributions."
+      ]
+    },
+    {
+      "metadata": {
+        "id": "TNGwb4TdrlJO",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "from __future__ import absolute_import\n",
+        "from __future__ import division\n",
+        "from __future__ import print_function\n",
+        "\n",
+        "import tensorflow as tf\n",
+        "import tensorflow_probability as tfp\n",
+        "import numpy as np\n",
+        "import matplotlib.pyplot as plt; plt.style.use('ggplot')\n",
+        "import matplotlib.cm as cm\n",
+        "from tqdm import tqdm\n",
+        "\n",
+        "tfd = tfp.distributions"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "-KdQXvGgrqVk",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "# Utility functions from:\n",
+        "# Original source in https://github.com/tensorflow/probability/blob/master/tensorflow_probability/examples/jupyter_notebooks/Bayesian_Gaussian_Mixture_Model.ipynb\n",
+        "\n",
+        "def session_options(enable_gpu_ram_resizing=True):\n",
+        "  \"\"\"Convenience function which sets common `tf.Session` options.\"\"\"\n",
+        "  config = tf.ConfigProto()\n",
+        "  config.log_device_placement = True\n",
+        "  if enable_gpu_ram_resizing:\n",
+        "    # `allow_growth=True` makes it possible to connect multiple colabs to your\n",
+        "    # GPU. Otherwise the colab malloc's all GPU ram.\n",
+        "    config.gpu_options.allow_growth = True\n",
+        "  return config\n",
+        "\n",
+        "def reset_sess(config=None):\n",
+        "  \"\"\"Convenience function to create the TF graph and session, or reset them.\"\"\"\n",
+        "  if config is None:\n",
+        "    config = session_options()\n",
+        "  tf.reset_default_graph()\n",
+        "  global sess\n",
+        "  try:\n",
+        "    sess.close()\n",
+        "  except:\n",
+        "    pass\n",
+        "  sess = tf.InteractiveSession(config=config)"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "ud3XnrsmCHrb",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "# For reproducibility\n",
+        "random = np.random.RandomState(seed=45)\n",
```

Comment:
I'd use rng (just to distinguish from a possible random module).

Reply:
I replaced random with rng as:
`rng = np.random.RandomState(seed=45)`